go/oasis-node/cmd/consensus/consensus.go

```diff
@@ -103,11 +110,43 @@ func doShowTx(cmd *cobra.Command, args []string) {
 	sigTx.PrettyPrint("", os.Stdout)
 }
 
+func doEstimateGas(cmd *cobra.Command, args []string) {
+	if err := cmdCommon.Init(); err != nil {
+		cmdCommon.EarlyLogAndExit(err)
+	}
+
```

Comment:
This is missing `cmdConsensus.InitGenesis()` which initializes the chain domain separation context so any signature verifications will fail. Adding it also means that you need to specify the path to the genesis file.